modules/subdyn/src/SD_FEM.f90

```diff
@@ -397,7 +402,7 @@ SUBROUTINE SD_ReIndex_CreateNodesAndElems(Init,p, ErrStat, ErrMsg)
          ! Test that the two properties match for non-beam 
          if (mType/=idMemberBeam) then
              if (Init%Members(iMem, iMProp)/=Init%Members(iMem, iMProp+1)) then
-                call Fatal('Properties should be the same at each node for non-beam members. Check member with ID: '//TRIM(Num2LStr(Init%Members(iMem,1))))
+                call Fatal('Property IDs should be the same at each joint for members with X section properties. Check member with ID: '//TRIM(Num2LStr(Init%Members(iMem,1))))
```

Comment:
It seems that this check was previously for anything that is not a beam (Rigid, Cable), and now we restrict it as well for anisotropic beams.  Shouldn't the text "with X section properties" be "for anisotropic beams, rigid links, and cables" maybe?

Reply:
Yes I didn’t realise this check was repeated twice as you have noted elsewhere.